libraries/AP_Scheduler/AP_Scheduler.h

```diff
@@ -200,6 +205,19 @@ class AP_Scheduler
 
     // bitmask bit which indicates if we should log PERF message
     uint32_t _log_performance_bit;
+
+    // maximum task slowdown compared to desired task rate before we
+    // start giving extra time per loop
+    const uint32_t max_task_slowdown = 8;
```

Comment:
uint32_t is somewhat overkill for a `const`.

Reply:
fixed